### PR TITLE
WFCORE-4868: implement aliases for standalone configuration files

### PR DESCRIPTION
**Pull request by michpetrov**
Issue: [WFCORE-4868](https://issues.redhat.com/browse/WFCORE-4868)

Should this be mentioned somewhere? I'm thinking `--help`, not user documentation.

### CODE REVIEW

File: controller/src/main/java/org/jboss/as/controller/persistence/ConfigurationFile.java [hunk 1]
```diff
@@ -362,6 +367,20 @@ private File determineMainFile(final String rawName, final String name) {
         throw ControllerLogger.ROOT_LOGGER.mainFileNotFound(name != null ? name : rawName, configurationDir);
     }
 
+    private String translateFileAlias(String alias) {
+        final int pos = rawFileName.lastIndexOf('.');
+        final String base = pos != -1 ? rawFileName.substring(0, pos) : rawFileName;
+        switch (alias) {
```
Comment (darranl) [inline controller/src/main/java/org/jboss/as/controller/persistence/ConfigurationFile.java:373]: Do we really want a list like this in the code, this becomes something else that we need to remember to maintain if the list of files changes.

If the argument does not specify a suffix could we not instead assume standalone-{argument}.xml and check if we have a match?

Comment (michpetrov) [reply controller/src/main/java/org/jboss/as/controller/persistence/ConfigurationFile.java:373]: I mean we could do that as well. The intention is not having to type long config names and e.g. `load-balancer` or 'microprofile` are IMHO still kinda long.

**Comment by darranl** [comment]
@michpetrov This PR has been flagged as a feature so should be following the RFE process, will you be progressing this as an RFE?

**Comment by michpetrov** [comment]
@darranl I am not familiar with the RFE process, what are the requirements?

**Comment by darranl** [comment]
This pull request is still active and awaiting approval to proceed.

**Comment by rsvoboda** [comment]
@michpetrov / @darranl what actions are remaining to get this PR merged?

**Comment by darranl** [comment]
@rsvoboda Continuation of the RFE process up until the point a pre-check has been granted.

**Comment by michpetrov** [comment]
@darranl well I wrote to the WildFly dev list twice and didn't get a reply, what do I do now?

**Pull request closed without merging**